### PR TITLE
Add spec for `Nylas::Event`

### PR DESCRIPTION
**Pull request by arunagw**
This PR adds basic spec for `Nylas::Event` class as these were
missing and also requested in #244

### CODE REVIEW

File: spec/nylas/event_spec.rb [hunk 1]
```diff
@@ -0,0 +1,152 @@
+# frozen_string_literal: true
+
+describe Nylas::Event do
+  describe ".from_json" do
```
Comment (adarsh) [inline spec/nylas/event_spec.rb:4]: Tiny nit: If it's an instance method, should this be `#from_json`?

File: spec/nylas/event_spec.rb [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+      api = instance_double(Nylas::API)
+      data = {
+        account_id: "acc-1234",
+        busy: true,
+        calendar_id: "cal-0987"
+      }
+
+      event = described_class.from_json(JSON.dump(data), api: api)
+
+      expect(event.busy?).to eq(true)
```
Comment (adarsh) [inline spec/nylas/event_spec.rb:82]: With RSpec matchers, you should be able to do 

```ruby
expect(event.busy?).to be_true
```
https://relishapp.com/rspec/rspec-expectations/v/2-2/docs/matchers/be-matchers

**Review by adarsh — APPROVED** [review]
Looks nice and clean! 👍

File: spec/nylas/event_spec.rb (hunk 1 above)
Comment (arunagw) [reply spec/nylas/event_spec.rb:4]: It's a class method as we are using it like `described_class.from_json`

File: spec/nylas/event_spec.rb (hunk 2 above)
Comment (arunagw) [reply spec/nylas/event_spec.rb:82]: We can even right a better way 

```
expect(event).to be_busy
```

File: spec/nylas/event_spec.rb (hunk 1 above)
Comment (adarsh) [reply spec/nylas/event_spec.rb:4]: Oh whoops, my mistake. There was an instance double so I thought it was an instance variable. Not a great assumption!

**Pull request merged**